packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -0,0 +1,46 @@
+import { type TSESLint, type TSESTree } from '@typescript-eslint/utils';
+
+import { createRule, getParserServices } from '../util';
+
+type MessageIds = 'forbiddenFunctionSpread';
+
+export default createRule<[], MessageIds>({
+  defaultOptions: [],
+  name: 'no-misused-spread',
+  meta: {
+    docs: {
+      description:
+        "Disallow spread operator that shouldn't be spread most of the time",
+      recommended: 'stylistic',
+      requiresTypeChecking: true,
+    },
+    messages: {
+      forbiddenFunctionSpread:
+        'Spreading a function is almost always a mistake. Did you forget to call the function?',
+    },
+    schema: [],
+    type: 'suggestion',
+  },
+  create: (context): TSESLint.RuleListener => {
+    const listener = (node: TSESTree.SpreadElement): void => {
```

Comment:
[Style] Is there a reason you'd wanted to split this out instead of having it inline at the bottom of the function? Inline would mean an inferrable type for `node`:

```ts
return {
  SpreadElement(node): void {
    // ...
  }
};
```